ui/service.go

```diff
@@ -1833,6 +1833,20 @@ func (us *uiService) ReadMessages(s Session, channelID, thingKey string, page, l
 	return btpl.Bytes(), nil
 }
 
+func (us *uiService) FetchChartData(token string, channelID string, mpgm sdk.MessagePageMetadata) ([]byte, error) {
+	msg, err := us.sdk.ReadMessages(mpgm, channelID, token)
```

Comment:
```suggestion
	msg, sdkErr := us.sdk.ReadMessages(mpgm, channelID, token)
```

The for json leave it as `err`